release-notes.md

```diff
@@ -0,0 +1,24 @@
+##Release Notes
+***
+* Date: 9 June, 2015
+* Product: Z-Manager Installer
+* Description: Initial release
```

Comment:
Do you think we need title "Description" ? 
At this point it might be obvious to th reader that text\bullet items below are actually a description